src/main/java/core/basesyntax/Application.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Application {
+
+    private static final Figure[] FIGURES = new Figure[new Random().nextInt(3, 6)];
+    private static final FigureSupplier FIGURE_SUPPLIER = new FigureSupplier();
```

Comment:
```suggestion
    private final FigureSupplier figureSupplier = new FigureSupplier();
```
No need to do static